### PR TITLE
Add subtract function

### PR DESCRIPTION
**Pull request by dfarr**

### CODE REVIEW

File: math.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 module.exports = {
   add: function(a, b) {
     return a + b;
+  },
+  
+  subtract: function(a, b) {
+    return a - b
```
Comment (gruiz17) [inline math.js]: missing semicolon !fix

Comment (dfarr) [inline math.js]: oops, will add right now...

Comment (gruiz17) [inline math.js]: looks good! **!fixed**

**Comment by gruiz17** [comment]
lol

**Comment by gruiz17** [comment]
hkhkjhjk

**Comment by gruiz17** [comment]
lajsdflkjdsaflkfdasjlkdfsj

**Comment by gruiz17** [comment]
ffffff

**Comment by gruiz17** [comment]
asdfadsfdas

**Comment by gruiz17** [comment]
jljljlkj

**Comment by gruiz17** [comment]
ffffff

**Comment by gruiz17** [comment]
s

**Comment by gruiz17** [comment]
foo

**Comment by gruiz17** [comment]
l

**Comment by dfarr** [comment]
test

**Pull request closed without merging**